RealmSwift/Realm.swift

```diff
@@ -300,7 +300,7 @@ public final class Realm {
     :nodoc:
     */
     @discardableResult
-    public func createDynamicObject(ofType typeName: String, populatedWith value: Any = [:], update: Bool = false) -> DynamicObject {
+    public func dynamicCreate(_ typeName: String, value: Any = [:], update: Bool = false) -> DynamicObject {
```

Comment:
This function's name _starts_ with `dynamic` to prevent it from showing up in autocorrect since it's not what users want 99% of the time. Ideally we'd have Clang submodules in Swift and expose it in there, but alas we don't, so we namespace it like this.